script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt

```diff
@@ -232,7 +232,7 @@ And right now, something similar,
 [sha:7e12b1b5fa1f9e2244eb9350e8d4468be00e0e9e]{
 -- Page 14, Offset 39386.
 -- 太陽が落ちたようだ。
-It's as if the sun had fallen to earth.
+It's as if the sun had fallen to the earth.
```

Comment:
I think original is fine?

Reply:
The new line sounded better in my head at first now not so much.